test/e2e/e2e-cli.sh

```diff
@@ -1,21 +1,27 @@
 #!/bin/sh
-
-npm run build
+set -e
```

Comment:
This should've been added from the very beginning and prevents script from continuing if a single step fails.